service/kvmclusterconfig/v1/resource/encryptionkey/update_test.go

```diff
@@ -50,10 +50,6 @@ func Test_newUpdateChange_Does_Not_Return_Change(t *testing.T) {
 				Logger:    logger,
 			})
 
-			if err != nil {
-				t.Fatalf("Resource construction failed: %#v", err)
-			}
```

Comment:
That does not look right. There is an error returned from the constructor and we should always handle any error.

Reply:
This is in unit test. The test cannot continue if there's no `Resource` providing implementation for test target. I can't see any other outcome here. Do you have suggestions?

Reply:
So you want your test to rely on some convention for return values in error cases? I would not do this. For me it is pretty simple. There is an error returned, we handle it. That is it. Why should we do anything different in any case? That just does not compute in my head.

Reply:
Oh gosh. Now I really see what happens here. I didn't notice the meaning of color here at all and I was really puzzled wondering what do you mean here. Now I notice that the error check is actually removed in this PR. A VERY good catch! Thank you! This is caused because I was developing the feature on parallel to that other Apply* PR and I was fixing feedback comments in both but for some reason after creating patch and applying to clean branch this sneaked in somehow. My apologies for wasted time here. I'll definitely put this back. This wasn't on purpose. :)